### PR TITLE
Feature/divider

### PR DESCRIPTION
**Pull request by ahmedkhitaby**
create divider atom

### CODE REVIEW

File: modules/shared/components/atoms/Divider/Divider.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import className from 'classnames';
+import type { FC, ReactElement, CSSProperties } from 'react';
+import type { IDivider } from './types/IDivider';
+import * as EDivider from './types/EDivider';
+
+const Divider: FC<IDivider.IProps> = ({ type, length }): ReactElement => {
+  const dividerType = className('bg-grey-shd6', {
+    'horizontal h-px': type === EDivider.DividerType.Horizontal,
```
Comment (Ali-Khamis) [inline modules/shared/components/atoms/Divider/Divider.tsx:9]: remove horizontal class as it does nothing

File: modules/shared/components/atoms/Divider/Divider.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    'vertical w-px': type === EDivider.DividerType.Vertical,
```
Comment (Ali-Khamis) [inline modules/shared/components/atoms/Divider/Divider.tsx:10]: remove vertical class as it does nothing

**Review by Ali-Khamis — CHANGES_REQUESTED** [review]

**Comment by ahmedkhitaby** [comment]
All requested changes have been updated @Ali-Khamis

**Review by Ali-Khamis — APPROVED** [review]
LGTM

**Pull request merged**